app/models/camp.rb

```diff
@@ -33,7 +33,11 @@ class Camp < ActiveRecord::Base
   validates_with CanCreateNewDreamValidator, :on => :create
 
   filterrific(
-    default_filter_params: { sorted_by: 'updated_at_desc' },
+    default_filter_params: {
+        sorted_by: 'updated_at_desc',
+        not_seeking_funding: 1,
```

Comment:
@sh0oki We didn't really used the parameter `not_seeking_funding` till now. did you test this?